scripts/bootstrapper/build.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# The script must be invoked with the concordium-node source directory as working directory.
+
+# Expected parameters:
+# - VERSION
+
+version="${VERSION}"
+
+# Install dependencies.
+
+apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install pkg-config
+
+# Build Debian packages.
+
+build_dir=/build
+p2p_bootstrapper_build_dir="$build_dir/p2p-bootstrapper_$version"
+p2p_bootstrapper_regenesis_block_hashes_file="/var/lib/concordium/data/genesis_hash"
+
+# - 'p2p-bootstrapper'
+
+(
+
+mkdir -p "$p2p_bootstrapper_build_dir"
+cd "$p2p_bootstrapper_build_dir"
+mkdir -p etc/default
+mkdir -p usr/lib/systemd/system
+mkdir -p var/lib/concordium/{data,config}
+mkdir -p usr/bin
+mkdir -p DEBIAN
+
+cat <<EOF > etc/default/p2p-bootstrapper
+CONCORDIUM_NODE_LISTEN_PORT=8888
+CONCORDIUM_NODE_CONFIG_DIR=/var/lib/concordium/config
+CONCORDIUM_NODE_DATA_DIR=/var/lib/concordium/data
+CONCORDIUM_NODE_PROMETHEUS_SERVER=true
+CONCORDIUM_NODE_PROMETHEUS_LISTEN_ADDRESS=0.0.0.0
+CONCORDIUM_NODE_PROMETHEUS_LISTEN_PORT=9090
+CONCORDIUM_NODE_LOG_LEVEL_DEBUG=true
+CONCORDIUM_NODE_NO_LOG_TIMESTAMP=true
+CONCORDIUM_NODE_BOOTSTRAPPER_REGENESIS_BLOCK_HASHES_FILE=$p2p_bootstrapper_regenesis_block_hashes_file
+EOF
+
+cat <<EOF > DEBIAN/control
+Package: p2p-bootstrapper
+Version: $version
+Section: extra
+Priority: optional
+Architecture: amd64
+Depends: debhelper ( >= 10 ), libunbound2 ( >= 1.6.7 ) | libunbound8 ( >= 1.9.4 ), openssl ( >= 1.1.1 ), libgmp10 ( >= 6.1.2 ), libpq-dev ( >= 9.5.19)
```

Comment:
We don't need libunbound and openssl and libpq I think.

Reply:
Cleaned up the dependencies in accordance with `ldd` output.